### Fixed

- The 500 / "The result file does not exist in export cache" error
  on dataset export request
  (<https://github.com/cvat-ai/cvat/pull/7864>)
